package.json

```diff
@@ -46,7 +51,10 @@
     "react-tagsinput": "^3.19.0"
   },
   "devDependencies": {
-    "@openshift-console/dynamic-plugin-sdk": "0.0.3",
+    "@cypress/webpack-preprocessor": "^5.9.1",
+    "cypress": "^8.3.1",
+    "cypress-multi-reporters": "^1.5.0",
+    "@openshift-console/dynamic-plugin-sdk": "0.0.4",
```

Comment:
@SanjalKatiyar In the older PR diff, I see that the version used was 0.0.4, but now its 0.0.3: https://github.com/red-hat-storage/mcg-ms-console/pull/6/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R48. Is this intentional? If so, I'll revert this change.

Reply:
Is the test failing because of this?